File: lab02/Python/fatorial.py
def fatorial(numero):    
    fat = 1
    i = 2
    while i <= numero:
        fat = fat*i
        i = i + 1

    print("O valor de %d! eh =" %numero, fat)

import math    
    
def fatorial_f(numero):
  return math.factorial(numero)

File: lab02/R/fatorial.R
fatorial <- function(numero) {
  fat <- 1
  i <- 2
  while (i <= numero) {
    fat = fat*i
    i = i + 1    
  }
  resultado = paste("O valor de", numero, "! eh =", fat)
  return(resultado)
}

fatorial_f <- function(numero) {
  resultado <- factorial(numero)
  return(resultado)
}
